types/openpgp/openpgp-tests.ts

```diff
@@ -135,6 +135,23 @@ openpgp.initWorker({ path:'openpgp.worker.js' });
     return verified.signatures[0].valid;
 })();
 
+async () => {
+    let hkp = new openpgp.HKP();
+
+    let hkpOptions = {
```

Comment:
I think we should use `const` instead. What do you think?

Reply:
good point, changed